### PR TITLE
Support Windows management server

### PR DESCRIPTION
**Pull request by jjohnson42**
Work to allow creation of standalone windows builds

### CODE REVIEW

File: confluent_server/confluent/log.py [hunk 1]
```diff
@@ -62,13 +62,21 @@
...
 import json
 import os
 import struct
 import time
 import traceback
 
+try:
+    from fcntl import flock, LOCK_EX, LOCK_UN, LOCK_SH
+except ImportError:
+    flock = lambda file, flag: True
```
Comment (chenglch) [inline confluent_server/confluent/log.py]: Just some suggestion. 
How about import msvcrt library if os.name == "nt"?
I think we would better add a common lock util to deal with the difference between posix and nt interface.

Comment (jjohnson42) [inline confluent_server/confluent/log.py]: Ok, that seems reasonable

**Comment by jjohnson42** [comment]
I have implemented windows lock...  can review last commit... will handle conflicts on monday

**Comment by jjohnson42** [comment]
Particularly I'm interested if you can review the changes to log and try them out.... 
May warrant a fix to the log rollover replay to chase textfile as well as bin file, but the pull request
isn't about that.

**Comment by whowutwut** [comment]
@jjohnson42 , just FYI,  China team is on holiday until Oct 12, I think Cheng Long will respond when he returns.

**Pull request merged**